internal/plugins/capacity_liquid.go

```diff
@@ -121,6 +139,29 @@ func (p *LiquidCapacityPlugin) Scrape(ctx context.Context, backchannel core.Capa
 		resultInService[resName] = resData
 	}
 
+	// manual capacity collection
+	fixedCapaConfig, fccExist := p.FixedCapacityConfiguration.Unpack()
+	if fccExist {
```

Comment:
For shortlived success variables like this, it's idiomatic to use short, undistinct names (usually `ok` or `exists`).
```suggestion
	fixedCapaConfig, ok := p.FixedCapacityConfiguration.Unpack()
	if ok {
```
Same below for `pccExist`.

Reply:
adressed